### ISSUE TITLE
No package matching '' is available

### ISSUE THREAD

**Issue by dbolotin** [haxorof/ansible-role-docker-ce#119]
### Version Information

Ansible: 2.9.10

Role: 2.6.5 **(downgrading to 2.6.4 resolved the issue)**

OS: Ubuntu 20.04

### Steps to Reproduce
```
- name: Install Docker
  include_role:
    name: haxorof.docker_ce
  vars:
    docker_sdk: yes
    docker_compose: yes
```

### Expected Behavior

Installs Docker CE

### Actual Behavior
```
TASK [haxorof.docker_ce : Install additional packages (OS package manager)] ************************************************************************************************************************************************************
FAILED - RETRYING: Install additional packages (OS package manager) (3 retries left).
FAILED - RETRYING: Install additional packages (OS package manager) (2 retries left).
FAILED - RETRYING: Install additional packages (OS package manager) (1 retries left).
failed: [my_host] (item=) => changed=false 
  ansible_loop_var: item
  attempts: 3
  item: ''
  msg: No package matching '' is available
ok: [my_host] => (item=['libffi-dev', 'libssl-dev'])

```

**Comment by haxorof** [comment]
Thanks for reporting this! Will look into this, might have introduced a bug when I did some refactoring.

**Comment by haxorof** [comment]
Manage to reproduce the error you got and got a fix for it. Did not test scenario where pip is already installed and missed this bug that I introduced. Will release a new version within the hour.

**Comment by dbolotin** [comment]
Thank you! Confirm that issue is fixed in 2.6.6.
